modes/longitudinal/src/toolbarButtons.js

```diff
@@ -326,12 +316,23 @@ const toolbarButtons = [
       type: 'tool',
       icon: 'tool-crosshair',
       label: 'Crosshairs',
+      // TODO - figure out what it takes here to deactivate crosshairs
+      volumeDeactivate: {
+        interactionType: 'tool',
+        commands: [
+          {
+            commandOptions: {
+              toolName: 'WindowLevel',
+            },
+            context: 'CORNERSTONE',
+          },
+        ],
+      },
       commands: [
         {
-          commandName: 'setToolActive',
           commandOptions: {
-            toolGroupId: 'mpr',
             toolName: 'Crosshairs',
+            toolGroupId: 'mpr',
           },
```

Comment:
where did `commandName : setToolActive` go?

Reply:
That is default for this one - could add it back if you want, but is implicit in being a tool

Reply:
I guess since we have it for everyone, either remove all or bring it back